services/movie.py

```diff
@@ -6,22 +7,27 @@
 def get_movies(
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
+    title: list[str] = None
 ) -> QuerySet:
-    queryset = Movie.objects.all()
+    queryset = Movie.objects.prefetch_related("Actor", "Genre")
```

Comment:
The `prefetch_related` method should use lowercase model names as strings, i.e., `"actors"` and `"genres"`, instead of `"Actor"` and `"Genre"`.